services/director-v2/src/simcore_service_director_v2/modules/dynamic_sidecar/scheduler/task.py

```diff
@@ -272,22 +274,18 @@ async def retrieve_service_inputs(
 
         service_name = self._inverse_search_mapping[node_uuid]
         scheduler_data: SchedulerData = self._to_observe[service_name]
-
+        dynamic_sidecar_endpoint: AnyHttpUrl = scheduler_data.dynamic_sidecar.endpoint
         dynamic_sidecar_client: DynamicSidecarClient = get_dynamic_sidecar_client(
             self.app
         )
 
-        transferred_bytes = await dynamic_sidecar_client.service_pull_input_ports(
-            dynamic_sidecar_endpoint=scheduler_data.dynamic_sidecar.endpoint,
-            port_keys=port_keys,
+        transferred_bytes = await dynamic_sidecar_client.ports_inputs_pull(
+            dynamic_sidecar_endpoint, port_keys
         )
 
         if scheduler_data.restart_policy == RestartPolicy.ON_INPUTS_DOWNLOADED:
             logger.info("Will restart containers")
-            await dynamic_sidecar_client.restart_containers(
-                scheduler_data.dynamic_sidecar.endpoint
-            )
-            logger.info("Containers restarted")
+            await dynamic_sidecar_client.containers_restart(dynamic_sidecar_endpoint)
```

Comment:
suggestion: ```restart_service```

Reply:
This one I tend not to agree. We restart the associated containers which are started by the sidecar. I would go at most for `restart_containers`

Reply:
from the point of view of director-v2 it's a service. but whatever. it's ok with container.